src/coinbase/wallet.ts

```diff
@@ -91,71 +117,33 @@ export class Wallet {
       },
     });
 
-    const wallet = await Wallet.init(result.data, undefined, []);
-
+    const wallet = await Wallet.init(result.data, undefined);
     if (Coinbase.useServerSigner) {
       await wallet.waitForSigner(wallet.getId()!, intervalSeconds, timeoutSeconds);
     }
 
     await wallet.createAddress();
-    await wallet.reload();
-
     return wallet;
   }
 
-  /**
-   * Waits until the ServerSigner has created a seed for the Wallet.
-   *
-   * @param walletId - The ID of the Wallet that is awaiting seed creation.
-   * @param intervalSeconds - The interval at which to poll the CDPService, in seconds.
-   * @param timeoutSeconds - The maximum amount of time to wait for the ServerSigner to create a seed, in seconds.
-   * @throws {APIError} if the API request to get a Wallet fails.
-   * @throws {Error} if the ServerSigner times out.
-   */
-  private async waitForSigner(
-    walletId: string,
-    intervalSeconds = 0.2,
-    timeoutSeconds = 20,
-  ): Promise<void> {
-    const startTime = Date.now();
-    while (Date.now() - startTime < timeoutSeconds * 1000) {
-      const response = await Coinbase.apiClients.wallet!.getWallet(walletId);
-      if (response?.data.server_signer_status === ServerSignerStatus.ACTIVE) {
-        return;
-      }
-      await delay(intervalSeconds);
-    }
-    throw new Error("Wallet creation timed out. Check status of your Server-Signer");
-  }
-
   /**
    * Returns a new Wallet object. Do not use this method directly. Instead, use User.createWallet or User.importWallet.
    *
    * @constructs Wallet
    * @param model - The underlying Wallet model object
    * @param seed - The seed to use for the Wallet. Expects a 32-byte hexadecimal with no 0x prefix. If null or undefined, a new seed will be generated.
    * If the empty string, no seed is generated, and the Wallet will be instantiated without a seed and its corresponding private keys.
-   * @param addressModels - The models of the addresses already registered with the Wallet. If not provided, the Wallet will derive the first default address.
    * @throws {ArgumentError} If the model or client is not provided.
    * @throws {InternalError} - If address derivation or caching fails.
    * @throws {APIError} - If the request fails.
    * @returns A promise that resolves with the new Wallet object.
    */
-  public static async init(
-    model: WalletModel,
-    seed?: string | undefined,
-    addressModels: AddressModel[] = [],
-  ): Promise<Wallet> {
-    this.validateSeedAndAddressModels(seed, addressModels);
-
+  public static async init(model: WalletModel, seed?: string | undefined): Promise<Wallet> {
```

Comment:
Does this function still need to be `async`? It doesn't seem like any of the functions within it are async any longer

Reply:
@alex-stone Good catch! I just removed `async` usage